tests/integration/changeset/changeset.spec.ts

```diff
@@ -57,7 +48,7 @@ describe('changeset', function () {
       it('should return 200 status code and OK body', async function () {
         const body = createStringifiedFakeChangeset();
 
-        await requestSender.postChangeset(app, body);
+        expectResponseStatusCode(await requestSender.postChangeset(app, body), StatusCodes.CREATED);
```

Comment:
It would be nicer if instead of a util function, we will extend jest with custom matcher.
you can see an example here:
https://github.com/MapColonies/osm-change-generator/blob/master/tests/matchers.js

Reply:
done.